File: app/src/main/java/com/example/weatherapp/ArpltnService.java
package com.example.weatherapp;

import io.reactivex.rxjava3.core.Observable;
import retrofit2.http.GET;
import retrofit2.http.Query;

interface ArpltnService {
    @GET("getMsrstnAcctoRltmMesureDnsty")
    Observable<ArpltnResult> getDnsty(@Query("serviceKey") String serviceKey,
                                      @Query("returnType") String returnType,
                                      @Query("numOfRows") String numOfRows,
                                      @Query("pageNo") String pageNo,
                                      @Query("dataTerm") String dataTerm,
                                      @Query("ver") String ver,
                                      @Query("stationName") String stationName);
}

File: app/src/main/java/com/example/weatherapp/data/source/remote/alpltn/ArpltnService.kt
package com.example.weatherapp.data.source.remote.alpltn

import io.reactivex.rxjava3.core.Observable
import retrofit2.http.GET
import retrofit2.http.Query

interface ArpltnService {
    @GET("getMsrstnAcctoRltmMesureDnsty")
    fun getDnsty(
        @Query("serviceKey") serviceKey: String?,
        @Query("returnType") returnType: String?,
        @Query("numOfRows") numOfRows: String?,
        @Query("pageNo") pageNo: String?,
        @Query("dataTerm") dataTerm: String?,
        @Query("ver") ver: String?,
        @Query("stationName") stationName: String?
    ): Observable<ArpltnResult>
}